corehq/apps/app_manager/suite_xml/case_tile_templates/one_two_two.xml

```diff
@@ -39,7 +39,7 @@
     </header>
     <template form="{header[format]}">
       <text>
-        **<locale id="{header[locale_id]}"/>:** ‎<!-- This line ends with a Left-to-right control character so the space after ** is preserved -->
```

Comment:
I wonder if we should leave just the left to right character in here so the AE does not have to add it manually.

Reply:
I've been looking for a simple way to keep the space but haven't been able to. For some reason I don't understand, when I keep the left to right character and input `**Name:**` as the display text, the markdown doesn't render and it displays `**Name:**`

Reply:
@Jtang-1 I think this is a limitation of the xml parsing as it trims the the values. Not sure if you can figure out where the trimming happens. My approach was to add the control character to prevent the trimming. If I remember correctly the space makes it all the way into the suite.xml (might want to confirm that) so the trimming probably happens in form player.

Reply:
Yes it does make it into the suite. Formplayer trims it because there's white space in between the new space and nodes, i.e between `<text>` and `<locale>`. There wasn't a straightforward way I could find to add a space on the formplayer side and I'm concerned with the blastzone of a change on the formplayer side.
```
    <header>
      <text>
        <locale id="{header[locale_id]}"/>
      </text>
    </header>
   ```
I added a FOUR-PER-EM SPACE U+2005 character which isn't trimmed and replicates the width of a space (1/4 em) [d8639bf](https://github.com/dimagi/commcare-hq/pull/33400/commits/d8639bf297679dd59f836c3589ebf2a152c298bd) so this will avoid the app builder from adding the space. I do like this more than the app builder having to manually add the space, so that's from bringing this up.